lib/apiservers/portlayer/restapi/handlers/task_handlers.go

```diff
@@ -257,14 +274,24 @@ func (handler *TaskHandlersImpl) WaitHandler(params tasks.WaitParams) middleware
 		return tasks.NewInspectInternalServerError().WithPayload(err)
 	}
 
+	// wait task to set started field to something
 	// wait task to set started field to something
 	err := task.Wait(&op, handle, params.Config.ID)
 	if err != nil {
-		op.Errorf("%s", err.Error())
-
-		return tasks.NewWaitInternalServerError().WithPayload(
-			&models.Error{Message: err.Error()},
-		)
+		switch err := err.(type) {
```

Comment:
I will mark them at the `Error` level since they are indications of an Error which should also be visible on standard deployments.

Reply:
and work on making them more concise. I wanted the logs to have an indicator as to what error case was seen in the handlers. Especially since the handler is the entrance to the portlayer.